DesignSafe-CI/portal
# task/WP-708: add unit tests and refactor getting toolbar rules into a method

## Overview: ##

This PR adds unit tests as a follow-up to [WP-707](https://tacc-main.atlassian.net/browse/WP-707)

Getting toolbar rules is refactored into a method to make unit testing easier.

## PR Status: ##


## Related Jira tickets: ##

* [WP-708](https://tacc-main.atlassian.net/browse/WP-708)

## Testing Steps: ##
1. See  testing instructions in [WP-707](https://tacc-main.atlassian.net/browse/WP-707)
2. Confirm unit tests make sense and run correctly.

## Notes

Do we want these simple file fixtures to be moved somewhere in `client/modules/_test-fixtures/src/fixtures`?
